xbitinfo/xbitinfo.py

```diff
@@ -439,7 +534,17 @@ def get_keepbits(info_per_bit, inflevel=0.99):
         # get only variables of bitdim
         bit_vars = [v for v in info_per_bit.data_vars if bitdim in info_per_bit[v].dims]
         if bit_vars != []:
-            cdf = _cdf_from_info_per_bit(info_per_bit[bit_vars], bitdim)
+            if information_filter == "On":
```

Comment:
Could you give the filter a name, like `gradient` so we can add additional filters in the future?
```suggestion
            if information_filter == "gradient":
```
You need to adapt this of course also at other places.